### DISCUSSION TITLE
Manually convert SqlAlchemy model to Pydantic model

### DISCUSSION THREAD

**Discussion by mdtomo** [fastapi/fastapi#4723] [Questions] [answered]
I am trying to manually convert a SqlAlchemly model to a Pydantic model in order to convert it to JSON with `jsonable_encoder` to send to a different API (AWS Lambda JSON payload).

I know that `APIRouter` does this automatically using the `response_model` but how can I manually do this? I was hoping there would be some kind of utility to do this.

I've looked at `APIRouter` to see how it works but can't find a practical way to use `run_endpoint_function` during a request.

Any help would be appreciated.

**Comment by followben** [comment, marked as answer]
You can convert from SQLAlchemy to Pydantic by setting `orm_mode` to `True` in your Pydantic class's config then using `from_orm()`. See https://pydantic-docs.helpmanual.io/usage/models/#orm-mode-aka-arbitrary-class-instances

To serialise a Pydantic instance use https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeljson

**Reply by mdtomo** [reply]
Thank you!

**Comment by LucasSovre** [comment]
You can also use 

```python
pydantic_model_instance = pydantic_model(**sql_alchemy_model_instance.__dict__) 
```
